src/pages/press/2020-11-09-civicactions-acquires-openconcept.js

```diff
@@ -10,15 +10,15 @@ const OpenConcept = () => {
     return (
         <RedLayout>
             <Helmet>
-                <title data-react-helmet="true">CivicActions expands accessibility capabilities with Open Concept</title>
+                <title data-react-helmet="true">CivicActions expands accessibility capabilities with OpenConcept</title>
             </Helmet>
             <Hero
                 title='CivicActions expands accessibility capabilities with Open Concept'
                 />
             <div className='single--press-releases--main'>
                 <div className='inner'>
                     <span className='body'>Date: Monday, November 9, 2020</span>
-                    <p className='body'><strong>Lafayette, Calif.</strong> – <a href=''>CivicActions</a>, a professional services firm modernizing government technologies and services, today announced it has acquired <a href='https://openconcept.ca/drupal'>OpenConcept</a>, a development firm specializing in web accessibility through open source tools and community.</p>
```

Comment:
In order that it does not make a new request for internal links, you can use the Link component. It should then go to the page near instantly when you click on the link.

Reply:
@alexiscott do you mean the <Link /> component ???